### PR TITLE
Mark fixes 2

### PR DESCRIPTION
**Pull request by mnunberg**

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
@mnunberg sorry I did not see that PR and I pushed some fixes to master. I will solve the conflicts.

File: RLTest/__main__.py [hunk 1]
```diff
@@ -231,6 +215,7 @@ def __init__(self):
             args = ['%s%s' % (RLTest_CONFIG_FILE_PREFIX, RLTest_CONFIG_FILE_NAME)] + sys.argv[1:]
         else:
             args = sys.argv[1:]
+        print args
```
Comment (MeirShpilraien) [inline RLTest/__main__.py]: remove?

File: RLTest/debuggers.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        cmd = ['valgrind', '--error-exitcode=1']
+        if self.leakcheck:
+            cmd += ['--leak-check=full', '--errors-for-leak-kinds=definite']
+        if self.suppressions:
+            cmd += ['--suppressions=' + self.suppressions]
+        if logfile:
+            cmd += ['--log-file' + logfile]
+        return cmd
+
+    def is_interactive(self):
```
Comment (MeirShpilraien) [inline RLTest/debuggers.py]: remove? as there is a var for this?

**Comment by MeirShpilraien** [comment]
Also the -t option is not working correctly, pls take a look.

**Comment by mnunberg** [comment]
@MeirShpilraien is `-t` still broken?

**Comment by mnunberg** [comment]
Fixed comments

**Comment by MeirShpilraien** [comment]
Now its not working to me at all, have you trying running the example file?

```
~/work/RedisSearch/RLTest > RLTest
	Unhandled exception: [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 480, in execute
    obj = test.create_instance()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/loader.py", line 32, in create_instance
    return self.cls(*args, **kwargs)
  File "/home/meir/work/RedisSearch/RLTest/test_example.py", line 11, in __init__
    self.env = Env()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/env.py", line 166, in __init__
    self.start()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/env.py", line 208, in start
    self.envRunner.startEnv()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/redis_std.py", line 150, in startEnv
    self.masterProcess = subprocess.Popen(args=self.masterCmdArgs, stdout=stdoutPipe, stderr=stderrPipe, stdin=stdinPipe, env=self.env)
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
test_example:test_example
	description: this is an example
	[ERROR]
	Unhandled exception: [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 403, in _runTest
    fn()
  File "/home/meir/work/RedisSearch/RLTest/test_example.py", line 32, in test_example
    env = Env(testDescription="this is an example")
  File "/home/meir/work/RedisSearch/RLTest/RLTest/env.py", line 166, in __init__
    self.start()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/env.py", line 208, in start
    self.envRunner.startEnv()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/redis_std.py", line 150, in startEnv
    self.masterProcess = subprocess.Popen(args=self.masterCmdArgs, stdout=stdoutPipe, stderr=stderrPipe, stdin=stdinPipe, env=self.env)
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/home/meir/.local/bin/RLTest", line 11, in <module>
    load_entry_point('RLTest', 'console_scripts', 'RLTest')()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 519, in main
    RLTest().execute()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 499, in execute
    self._runTest(test)
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 440, in _runTest
    return numFailed
UnboundLocalError: local variable 'numFailed' referenced before assignment
```

**Comment by MeirShpilraien** [comment]
Also this :
```
~/work/RedisSearch/RLTest > RLTest --env-only --env oss-cluster
Traceback (most recent call last):
  File "/home/meir/.local/bin/RLTest", line 11, in <module>
    load_entry_point('RLTest', 'console_scripts', 'RLTest')()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 519, in main
    RLTest().execute()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/__main__.py", line 461, in execute
    env = Env(testName='manual test env')
  File "/home/meir/work/RedisSearch/RLTest/RLTest/env.py", line 159, in __init__
    self.envRunner = self.getEnvByName()
  File "/home/meir/work/RedisSearch/RLTest/RLTest/env.py", line 205, in getEnvByName
    **kwargs)
TypeError: __init__() got an unexpected keyword argument 'debugger'
```

**Comment by mnunberg** [comment]
Let me try the example...

**Comment by mnunberg** [comment]
Ok, fixed!

**Pull request merged**